src/ram/src/ram.cpp

```diff
@@ -480,27 +444,27 @@ void RamGen::generate(const int bytes_per_word,
     array<dbBTerm*, 8> D;     // array for b-term for external inputs
     array<dbNet*, 8> D_nets;  // net for buffers
     for (int bit = 0; bit < 8; ++bit) {
-      D[bit] = makeBTerm(fmt::format("D[{}]", bit + col * 8));
+      D[bit] = makeBTerm(fmt::format("D[{}]", bit + col * 8), dbIoType::INPUT);
       D_nets[bit] = makeNet(fmt::format("D_nets[{}]", bit + col * 8), "net");
     }
 
     vector<array<dbBTerm*, 8>> Q;
```

Comment:
The name on this is tough because it represents Q bterms of multiple ports. Maybe instead name it `read_port_bterms`?